src/bindings/python/src/compatibility/pyngraph/util.cpp

```diff
@@ -42,4 +44,8 @@ void regmodule_pyngraph_util(py::module m) {
         return get_ngraph_version_string();
         NGRAPH_SUPPRESS_DEPRECATED_END
     });
+
+    mod.def("get_ie_output_name", [](const ngraph::Output<ngraph::Node>& output) {
```

Comment:
We didn't have such API in C++ for the end user. Why should we have it here?

Reply:
This lets me use the C++ helper in Python test runner. It's not exposed to the end user though (by __init__.py files).
If I don't expose this helper, I have to rewrite it in Python.